UnityProject/Assets/Scripts/Weapons/Projectiles/Behaviours/ProjectileDamageLivingHealth.cs

```diff
@@ -28,11 +30,13 @@ public bool OnHit(RaycastHit2D hit)
 
 		private bool TryDamage(RaycastHit2D hit)
 		{
+			var newDamage = damageData.Damage;
 			var coll = hit.collider;
 			var livingHealth = coll.GetComponent<LivingHealthBehaviour>();
 			if (livingHealth == null) return false;
+		
 
-			livingHealth.ApplyDamageToBodypart(shooter, damageData.Damage, damageData.AttackType, damageData.DamageType, targetZone);
+			livingHealth.ApplyDamageToBodypart(shooter, newDamage, damageData.AttackType, damageData.DamageType, targetZone);
```

Comment:
since you're not doing any further calculations, just use ``damageData.Damage``.